src/ADTs/queue.h

```diff
@@ -50,7 +50,7 @@ struct Queue {
 };
 
 /* Create empty Queue */
-struct Queue *init_queue();
+struct Queue *init_queue(void);
```

Comment:
@OrkoHunter This is cool.
I read the coding style guide but overlooked this.
Nice !